src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -327,26 +327,22 @@ public bool TryGetTemplateFromConfigInfo(IFileSystemInfo templateFileConfig, out
             }
             else if (param.IsChoice())
             {
-                if (TryResolveChoiceValue(literal, param, out string? match))
+                if (param.AllowMultipleValues)
                 {
-                    return match;
-                }
+                    List<string?> val = literal.Tokenize().Select(t => ResolveChoice(environmentSettings, t, param)).Where(r => !string.IsNullOrEmpty(r)).ToList();
```

Comment:
(not actionable) 
it would be great to move from `string, string` relation to `string, object` or `string, IParameterValue` relation for template parameters as part of the further refactoring

Reply:
Yeah - that would be much better. However we still likely would need to keep it in edge contracts